src/Command/Argument/Hash/CommonExpiration.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+/*
+ * This file is part of the Predis package.
+ *
+ * (c) 2009-2020 Daniele Alessandri
+ * (c) 2021-2023 Till Krüss
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Predis\Command\Argument\Hash;
+
+use Predis\Command\Argument\ArrayableArgument;
+use UnexpectedValueException;
+
+abstract class CommonExpiration implements ArrayableArgument
+{
+    /**
+     * @var array
+     */
+    protected $expireModifierEnum = [
+        'NX', 'XX', 'GT', 'LT',
+    ];
+
+    /**
+     * @var array
+     */
+    protected $ttlModifierEnum = [
+        'EX', 'PX', 'EXAT', 'PXAT',
+    ];
+
+    /**
+     * @var array
+     */
+    protected $arguments = [];
+
+    /**
+     * Set the modifier that defines a behaviour on expiration.
+     *
+     * NX for each specified field set expiration only when the field has no expiration
```

Comment:
Being picky here, missing end dot at two sentences. And some more at the other method below.

Reply:
Got it